ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ShowPopup/ShowPopup.cpp

```diff
@@ -29,7 +29,8 @@ using namespace Esri::ArcGISRuntime;
 
 ShowPopup::ShowPopup(QObject* parent /* = nullptr */):
   QObject(parent),
-  m_map(new Map(QUrl("https://runtime.maps.arcgis.com/home/webmap/viewer.html?webmap=e4c6eb667e6c43b896691f10cc2f1580"), this))
+  m_map(new Map(QUrl("https://runtime.maps.arcgis.com/home/webmap/viewer.html?webmap=e4c6eb667e6c43b896691f10cc2f1580"), this)),
+  m_popupManagers(new QList<PopupManager*>)
```

Comment:
This will leak memory, but that is beside the point since it should not be a pointer.